src/backend/binaryen/utils.ts

```diff
@@ -1774,6 +1774,10 @@ export namespace FunctionalFuncs {
             dyntype.dyn_ctx_t,
         );
         switch (opKind) {
+            case ts.SyntaxKind.LessThanEqualsToken:
+            case ts.SyntaxKind.LessThanToken:
+            case ts.SyntaxKind.GreaterThanEqualsToken:
+            case ts.SyntaxKind.GreaterThanToken:
```

Comment:
I think these `compare-kind` tokens can put just before `default`, they acts not like the `equal-kind` token.

Reply:
done